pkg/collector/corechecks/embed/jmxloader.go

```diff
@@ -21,6 +21,15 @@ import (
 
 var JMXConfigCache = cache.NewBasicCache()
 
+func AddJMXCachedConfig(config check.Config) {
```

Comment:
This function is exported so could you please add a comment for the godoc describing its (obvious) behavior and the fact it's a thread-safe call?